src/it/scala/io/iohk/ethereum/sync/util/SyncCommonItSpecUtils.scala

```diff
@@ -0,0 +1,51 @@
+package io.iohk.ethereum.sync.util
+
+import java.util.concurrent.{ThreadLocalRandom, TimeoutException}
+
+import io.iohk.ethereum.network.PeerManagerActor.FastSyncHostConfiguration
+import monix.eval.Task
+
+import scala.concurrent.duration.FiniteDuration
+
+object SyncCommonItSpecUtils {
+  def retryUntilWithDelay[A](source: Task[A], delay: FiniteDuration, maxRetries: Int)(
+    predicate: A => Boolean
+  ): Task[A] = {
+    source.delayExecution(delay).flatMap { result =>
+      if (predicate(result)) {
+        Task.now(result)
+      } else {
+        if (maxRetries > 0) {
+          retryUntilWithDelay(source, delay, maxRetries - 1)(predicate)
+        } else {
+          Task.raiseError(new TimeoutException("Task time out after all retries"))
+        }
+      }
+    }
+  }
+
+  case class HostConfig(
+                         maxBlocksHeadersPerMessage: Int,
+                         maxBlocksBodiesPerMessage: Int,
+                         maxReceiptsPerMessage: Int,
+                         maxMptComponentsPerMessage: Int
+                       ) extends FastSyncHostConfiguration
+
```

Comment:
IMHO sbt scalafmtAll will format this a bit differently. Probably this code is not updated by recent develop changes.
(But it is worth to check if `it` scope will be affected)

Reply:
Done,
merge with develop, but I think that "it" scope dont be affected.

Reply:
we should probably enable formatting in all sub projects. 

You can do that by changing:
`.settings(inConfig(Integration)(Defaults.testSettings :+ (Test / parallelExecution := false)): _*)` to
`    .settings(inConfig(Integration)(Defaults.testSettings ++ (org.scalafmt.sbt.ScalafmtPlugin.scalafmtConfigSettings) :+ (Test / parallelExecution := false)): _*)`

in `build.sbt`.

Reply:
Done